week1/day_4_plotting_exercises.R

```diff
@@ -0,0 +1,109 @@
+# Session 3.3.1, exercise 1, 2, and 3
+library(tidyverse)
+
+ggplot(data = mpg) + 
+  geom_point(mapping = 
+               aes(x = displ, y = hwy, color = "blue"))
+
+# 1. The points aren't blue because the argument color is a string blue that is treated as an aes. 
+# This should be a mapping between a variable and a value. "blue" is not a value that represents the color blue. 
+# If you place, color = "blue" outside the aes, they the graph would be blue. 
+
+# 2. categorical variables: manufacturer, model, trans, drv, fl, class
+# continuous variables are: displ, year, cyl, cty, hey
+
+# 3. 
+ggplot(data = mpg) + 
+  geom_point(mapping = 
+               aes(x = displ, y = hwy, color = year))
+# a continuous variable for color uses a scale of colors from light to dark.
+ggplot(data = mpg) + 
+  geom_point(mapping = 
+               aes(x = displ, y = hwy, size = year))
+# a continuous variable for size, changes how big or small the points on the graph are. 
+ggplot(data = mpg) + 
+  geom_point(mapping = 
+               aes(x = displ, y = hwy, shape = year))
+# A continuous variable can not be mapped to shape.
+# I think the color and size are not the best to use for continuous variables. 
+# The reader of the graph needs to keep on going back to the legend to understand the graph. 

```

Comment:
I agree with you in general but there are some good cases. Like for example we might want to do a political spectrum and it would work well with a continuous color scale from blue to red for example.